app/legacy_lib/query_supporters.rb

```diff
@@ -641,6 +641,19 @@ def self.dupes_on_last_name_and_address(np_id)
       .map { |arr_group| arr_group.flatten.sort }
   end
 
+  def self.dupes_on_last_name_and_address_and_email(np_id)
+    dupes_expr(np_id)
+      .and_where(
+        "name IS NOT NULL\
+         AND name != ''\
+         AND address IS NOT NULL \
+         AND address != ''"
+      )
+      .group_by(self.calculated_last_name +  " || '_____' || address || '_____' || COALESCE(email, '')")
+      .execute(format: 'csv')[1..-1]
```

Comment:
Based on the code in https://github.com/CommitChange/houdini/blob/cf4c27c9ddd63619b23544326a400c3930d89c79/gems/ruby-qx/lib/qx.rb#L138-L157, it appears `format: 'csv'` means "return an array of arrays". Removing the first element is because the first item of the array is normally the header row and that's not useful in this case.